va/va_vpp.h

```diff
@@ -456,10 +584,14 @@ typedef struct _VAProcPipelineParameterBuffer {
      */
     uint32_t        output_background_color;
     /**
-     * \brief Requested output color primaries.
+     * \brief Requested output color standard.
      */
     VAProcColorStandardType output_color_standard;
     /**
+     * \brief Requested output color properties.
+     */
+    VAProcColorStandardType output_color_properties;
```

Comment:
VAProcColorProperties I guess.

@kodabb might chime in regarding the need of more fields.